README.md

```diff
@@ -16,12 +16,6 @@ You can optionally add a bit of extra structure to your internal packages to sep
 
 Library code that's ok to use by external applications (e.g., `/pkg/mypubliclib`). Other projects will import these libraries expecting them to work, so think twice before you put something here.
 
-### `/vendor`
-
```

Comment:
This isn't true anymore?

Reply:
Oh, I guess this is just taken care of by the make commands now?

Reply:
Vendoring third-party libs is apparently an antipattern for shared modules. I think it's only recommended for application codebases.

Reply:
Could you link some documentation on that? Curious why that would be since this repo should stand/build on its own.

Reply:
I've actually had trouble tracking down documentation on this, but the go layout template I used for this project actually states it here https://github.com/golang-standards/project-layout/tree/master/vendor:

> Don't commit your application dependencies if you are building a library.

I'm not clear on what happens with the `vendor` directory in a third-party lib, but I'm guessing it's not used by a consuming application. So, if we're vendoring our code for the purpose of testing, we could have passing builds while some consumer's build would fail due to some other third-party issue. In addition, the consuming application's dependencies may end up with some conflicts with ours.